packages/box/test/mock-local-box/truffle-box.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Could you put the mock project in a `sources` folder? It is a convention in the Truffle codebase to include all fixtures in a `sources` folder in the `test` folder.